diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index d1237542591ecc43143d067ce1437567698615ec..0000000000000000000000000000000000000000
diff --git a/dist/medium-zoom.esm.js b/dist/medium-zoom.esm.js
index 050c49208302fe510fe3a4d66b203b1d83dbc902..9e624e1beec684c90065b3bf516bc44a40fd4ba6 100644
--- a/dist/medium-zoom.esm.js
+++ b/dist/medium-zoom.esm.js
@@ -625,7 +625,7 @@ function styleInject(css, ref) {
   }
 }
 
-var css = ".medium-zoom-overlay{position:fixed;top:0;right:0;bottom:0;left:0;opacity:0;transition:opacity .3s;will-change:opacity}.medium-zoom--opened .medium-zoom-overlay{cursor:pointer;cursor:zoom-out;opacity:1}.medium-zoom-image{cursor:pointer;cursor:zoom-in;transition:transform .3s cubic-bezier(.2,0,.2,1)!important}.medium-zoom-image--hidden{visibility:hidden}.medium-zoom-image--opened{position:relative;cursor:pointer;cursor:zoom-out;will-change:transform}";
+var css = ".medium-zoom-overlay{position:fixed;top:0;right:0;bottom:0;left:0;opacity:0;z-index:40;transition:opacity .3s;will-change:opacity}.medium-zoom--opened .medium-zoom-overlay{cursor:pointer;cursor:zoom-out;opacity:1}.medium-zoom-image{cursor:pointer;cursor:zoom-in;transition:transform .3s cubic-bezier(.2,0,.2,1)!important}.medium-zoom-image--hidden{visibility:hidden}.medium-zoom-image--opened{position:relative;cursor:pointer;cursor:zoom-out;z-index:50;will-change:transform}";
 styleInject(css);
 
 export default mediumZoom;
diff --git a/dist/medium-zoom.js b/dist/medium-zoom.js
index 0ec46816014b9237c1043895dffb154b01966a4b..c05e1daeb40ba418e0ff2a30dd328603e0cd0c4d 100644
--- a/dist/medium-zoom.js
+++ b/dist/medium-zoom.js
@@ -478,7 +478,7 @@
       style.appendChild(document.createTextNode(css));
     }
   }
-  var css = ".medium-zoom-overlay{position:fixed;top:0;right:0;bottom:0;left:0;opacity:0;transition:opacity .3s;will-change:opacity}.medium-zoom--opened .medium-zoom-overlay{cursor:pointer;cursor:zoom-out;opacity:1}.medium-zoom-image{cursor:pointer;cursor:zoom-in;transition:transform .3s cubic-bezier(.2,0,.2,1)!important}.medium-zoom-image--hidden{visibility:hidden}.medium-zoom-image--opened{position:relative;cursor:pointer;cursor:zoom-out;will-change:transform}";
+  var css = ".medium-zoom-overlay{position:fixed;top:0;right:0;bottom:0;left:0;opacity:0;z-index:40;transition:opacity .3s;will-change:opacity}.medium-zoom--opened .medium-zoom-overlay{cursor:pointer;cursor:zoom-out;opacity:1}.medium-zoom-image{cursor:pointer;cursor:zoom-in;transition:transform .3s cubic-bezier(.2,0,.2,1)!important}.medium-zoom-image--hidden{visibility:hidden}.medium-zoom-image--opened{position:relative;cursor:pointer;cursor:zoom-out;z-index:50;will-change:transform}";
   styleInject(css);
   return mediumZoom;
 });
diff --git a/dist/style.css b/dist/style.css
index b5e0d320deaef0c4584b8810e9c4f211d8252ef7..332648ae0fb336118b0952d7d9cc627834ea2878 100644
--- a/dist/style.css
+++ b/dist/style.css
@@ -1 +1 @@
-.medium-zoom-overlay{position:fixed;top:0;right:0;bottom:0;left:0;opacity:0;transition:opacity .3s;will-change:opacity}.medium-zoom--opened .medium-zoom-overlay{cursor:pointer;cursor:zoom-out;opacity:1}.medium-zoom-image{cursor:pointer;cursor:zoom-in;transition:transform .3s cubic-bezier(.2,0,.2,1)!important}.medium-zoom-image--hidden{visibility:hidden}.medium-zoom-image--opened{position:relative;cursor:pointer;cursor:zoom-out;will-change:transform}
\ No newline at end of file
+.medium-zoom-overlay{position:fixed;top:0;right:0;bottom:0;left:0;opacity:0;z-index:40;transition:opacity .3s;will-change:opacity}.medium-zoom--opened .medium-zoom-overlay{cursor:pointer;cursor:zoom-out;opacity:1}.medium-zoom-image{cursor:pointer;cursor:zoom-in;transition:transform .3s cubic-bezier(.2,0,.2,1)!important}.medium-zoom-image--hidden{visibility:hidden}.medium-zoom-image--opened{position:relative;cursor:pointer;cursor:zoom-out;z-index:50;will-change:transform}
